presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java

```diff
@@ -533,14 +538,14 @@ public void testRound()
         assertFunction("round(0.3)", DOUBLE, 0.0);
         assertFunction("round(-0.3)", DOUBLE, -0.0);
 
-        assertFunction("round(FLOAT '3.0', 0)", DOUBLE, (double) 3.0f);
-        assertFunction("round(FLOAT '-3.0', 0)", DOUBLE, (double) -3.0f);
-        assertFunction("round(FLOAT '3.499', 0)", DOUBLE, (double) 3.0f);
-        assertFunction("round(FLOAT '-3.499', 0)", DOUBLE, (double) -3.0f);
-        assertFunction("round(FLOAT '3.5', 0)", DOUBLE, (double) 4.0f);
-        assertFunction("round(FLOAT '-3.5', 0)", DOUBLE, (double) -4.0f);
-        assertFunction("round(FLOAT '-3.5001', 0)", DOUBLE, (double) -4.0f);
-        assertFunction("round(FLOAT '-3.99', 0)", DOUBLE, (double) -4.0f);
+        assertFunction("round(FLOAT '3.0', 0)", FLOAT, 3.0f);
+        assertFunction("round(FLOAT '-3.0', 0)", FLOAT, -3.0f);
+        assertFunction("round(FLOAT '3.499', 0)", FLOAT, 3.0f);
+        assertFunction("round(FLOAT '-3.499', 0)", FLOAT, -3.0f);
+        assertFunction("round(FLOAT '3.5', 0)", FLOAT, 4.0f);
+        assertFunction("round(FLOAT '-3.5', 0)", FLOAT, -4.0f);
```

Comment:
This will need to be changed when merging with master since https://github.com/prestodb/presto/commit/de64398183d3394c163022a60fa83c2311b45d78 changes behavior.